components/src/jsMain/kotlin/dev/fritz2/components/alert.kt

```diff
@@ -163,7 +164,7 @@ open class AlertComponent : Component<Unit> {
                     }
                 }
 
-                box(baseClass = alertContentCss) {
+                div(baseClass = alertContentCss.name) {
```

Comment:
Imho wrong! We need a ``StyleClass`` there and not a ``String`` - as the class also gets applied, I would definitly consider this as false correction!

Reply:
Ok, I revoke my comment!

This is for sure something off within our API... the overloaded factories have a different type of course, as the styling DSL (and therefor the type ``StyleClass``) is not available at core level!